rubrik_cdm/cluster.py

```diff
@@ -20,9 +20,79 @@
 
 
 class Cluster(Api):
-    """This class contains methods related to the managment of the Rubrik cluster itself.
+    """This class contains methods related to the management of the Rubrik cluster itself.
     """
 
+    def configure_replication_private(self, username, password, target_ip,
+                              ca_certificate=None, timeout=30):
+        """Configure replication partner as specified by user using PRIVATE NETWORK (direct connection)
+        Arguments:
+            username {str} -- Username for the TARGET cluster
+            password {str} -- Password for the TARGET cluster
+            target_ip {str} -- Address of one of the nodes of the TARGET cluster {string}
+        Keyword Arguments:
+            ca_certificate {str} -- CA certificiate used to perform TLS certificate validation (default: {None})
+            timeout {int} -- The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error. (default: {30})
+        Returns:
+            dict -- The full API response from `POST /internal/replication/target`.
+        """
+
+        config = {}
+
+        config['replicationSetup'] = 'Private Network'
+        config['targetClusterAddress'] = target_ip
+        config['username'] = username
+        config['password'] = password
+
+        if ca_certificate is not None:
+            config["caCerts"] = ca_certificate
+
+        self.log("configure_replication: Adding cluster '{}' as private network replication target.".format(target_ip))
+
+        return self.post("internal", "/replication/target", config, timeout)
+
+    def configure_replication_nat(self, username, password, source_gateway, target_gateway,
+                              ca_certificate=None, timeout=30):
+        """Configure replication partner as specified by user using NAT
+        Arguments:
+            username {str} -- Username for the TARGET cluster {string}
+            password {str} -- Password for the TARGET cluster {string}
+            source_gateway {list} -- Specification of source NAT gateway specified as [str IP, [list of portnumber(s)]]
+            target_gateway {list} -- Specification of source NAT gateway specified as [str IP, [list of portnumber(s)]]
```

Comment:
`[str IP, [list of portnumber(s)]]`  .... someone who is just getting started with programming may not know what "str" means here. Probably can just do `["IP", [list of portnumber(s)]]`